docs/src/network/miden-client.md

```diff
@@ -0,0 +1,132 @@
+# Miden Client
+# Miden client
+
+The Miden client provides a way to execute and prove transactions, facilitating the interaction with the Miden rollup.
+
+### Status
+
+The Miden client is still under heavy development and the project can be considered to be in an *alpha* stage. Many features are yet to be implemented and there is a number of limitations which we will lift in the near future. We develop the Miden client in [this repo](https://github.com/0xPolygonMiden/miden-client/tree/main).
+
+## Overview
+
+The client's main responsibility is to maintain a partial view of the blockchain which allows for locally executing and proving transactions. It keeps a local store of various entities that periodically get updated by communicating with the node.
```

Comment:
Would change to: It stores various entities.

And also explain what these entities are.